### DISCUSSION TITLE
AMReX plotfile data to MATLAB

### DISCUSSION THREAD

**Discussion by HardeoC** [AMReX-Codes/amrex#3313] [Q&A]
Does anyone know where I can find routines to bring AMReX plotfiles into MATLAB to post-processs them there?

**Comment by WeiqunZhang** [comment]
I am not a MATLAB user. https://github.com/AMReX-Codes/amrex/blob/development/Tools/Postprocessing/C_Src/PlotfileToMatLab.cpp might do what you need.
